internal/command/container.go

```diff
@@ -197,7 +197,7 @@ func fyneCommand(command string, ctx Context, image containerImage) ([]string, e
 		"-appVersion", ctx.AppVersion,
 	}
 
-	if ctx.Package != "." {
+	if ctx.Package != "." && image.OS() != androidOS {
```

Comment:
Should this note be a check for mobile rather than Android? iOS follows the same build constraints in this regard does it not?

Reply:
Actually not, as the iOS build are not going through the container build, but are straight host command (and are located in command.go).

Reply:
Might that ever change? The reason this path is needed is because of a problem with our go-mobile adopted code - not the Android specific builder...

Reply:
I don't think we have any plan or even idea on how we could do the ios build in the container at this point. But I realized that I actually refactored the code too much and that the container and the command file shared that code which was incorrect. Fixed now.